projects/hipdnn/backend/src/FlatbufferUtilities.hpp

```diff
@@ -7,12 +7,9 @@
 #include <hipdnn_sdk/data_objects/graph_generated.h>
 #include <memory>
 
-namespace hipdnn_backend
-{
-namespace flatbuffer_utilities
+namespace hipdnn_backend::flatbuffer_utilities
 {
 void convertSerializedGraphToGraph(const uint8_t* buffer,
                                    size_t size,
                                    std::unique_ptr<hipdnn_sdk::data_objects::GraphT>& graphOut);
 }
```

Comment:
```suggestion
} // namespace hipdnn_backend::flatbuffer_utilities
```
Nit, but for consistency.